ITE/3/README.adoc

```diff
@@ -0,0 +1,596 @@
+= ITE-3: Real-world example of combining TUF and in-toto for packaging Datadog Agent integrations
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 5
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 3
+
+| Title
+| in-toto Enhancement format
+
+| Sponsor
+| link:https://github.com/trishankatdatadog[Trishank Karthik Kuppusamy]
+
+| Status
+| Draft :pensive:
+
+| Type
+| Informational
+
+| Created
+| 2019-07-19
+
+|===
+
+[[abstract]]
+== Abstract
+
+Datadog is a monitoring service for cloud-scale applications that monitors
+servers, databases, tools, and services through a
+software-as-a-service-based data analytics platform. It supports
+multiple cloud service providers, including Amazon Web Services,
+Microsoft Azure, Google Cloud Platform, and Red Hat OpenShift. At the
```

Comment:
This last sentence feels a little too much like an advertisement and will likely change over time (hopefully growing!).  Would it be okay to remove?